### DISCUSSION TITLE
How to get approximate orbit positions in kinematic POD for LEO satellites?

### DISCUSSION THREAD

**Discussion by whulizhen** [groops-devs/groops#194] [General]
Hi everyone
When I did kinematic POD for a LEO satellite (its not GRACE or any other LEO satellies in the example), it seems I need to input the approximate positions in the "receiver" section of the program "GnssProcessing". As this is not a GRACE or LEO satellites in the example, I cannot use GraceL1b2Orbit, Groops2Orbit or other programs to get the approximate orbit file. Dose anyone knows how can I get the approximate positions for any Cubesat?

**Comment by zhedumi** [comment]
Hi,
Yeah for the receiver and for LEO you need an apriori orbit the model can be fitted to. You could preprocess the orbit in a similar way as described within [LEO kinematicOrbit processing](https://groops-devs.github.io/groops/html/cookbook.kinematicOrbit.html). I think it this cookbook should contian all necessary steps to preprocess and process LEO satellites.

Best regards,
Patrick
